WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -88,72 +89,41 @@ final class OrderDetailsResultsControllers {
 
     /// Order shipment tracking list
     ///
-    var orderTracking: [ShipmentTracking] {
-        return trackingResultsController.fetchedObjects
-    }
+    private(set) var orderTracking: [ShipmentTracking] = []
```

Comment:
for this and similar changes in this file, are they required for resolving the hangs performance warnings? If not, maybe it's safer not changing them to keep the PR changes more isolated and prevent any unexpected issues.

Reply:
Reverted in 2dfb6accd1e4602aaea643adff5a0e829d1e7b7e.